zeek_log_transport.sh

```diff
@@ -15,7 +15,117 @@
 
 export PATH="/sbin:/usr/sbin:$PATH"		#Note that cron does _NOT_ include /sbin in the path, so attempts to locate the "ip" binary fail without this fix
 
-default_user_on_aihunter='dataimport'
+default_user_on_aihunter="dataimport"
+
+get_send_logs() {
+	##These names were found at https://docs.zeek.org/en/master/script-reference/log-files.html
+	declare -a zeek_logs=(	"barnyard2"
+				"broker"
+				"capture_loss"
+				"cluster"
+				"config"
+				"conn"
+				"conn-summary"
+				"dce_rpc"
+				"dhcp"
+				"dnp3"
+				"dns"
+				"dpd"
+				"files"
+				"ftp"
+				"http"
+				"intel"
+				"irc"
+				"kerberos"
+				"known_certs"
+				"known_hosts"
+				"known_modbus"
+				"known_services"
+				"loaded_scripts"
+				"modbus"
+				"modbus_register_change"
+				"mysql"
+				"netcontrol"
+				"netcontrol_catch_release"
+				"netcontrol_drop"
+				"netcontrol_shunt"
+				"notice"
+				"notice_alarm"
+				"ntlm"
+				"ntp"
+				"ocsp"
+				"openflow"
+				"packet_filter"
+				"pe"
+				"print"
+				"prof"
+				"radius"
+				"rdp"
+				"reporter"
+				"rfb"
+				"signatures"
+				"sip"
+				"smb_cmd"
+				"smb_files"
+				"smb_mapping"
+				"smtp"
+				"snmp"
+				"socks"
+				"software"
+				"ssh"
+				"ssl"
+				"stats"
+				"stderr"
+				"stdout"
+				"syslog"
+				"traceroute"
+				"tunnel"
+				"unified2"
+				"unknown_protocols"
+				"weird"
+				"weird_stats"
+				"x509"
+			)
```

Comment:
It's not often this would happen, but adding a new log file would mean having to update this list and thus the installed script (which is what we're trying to avoid having to do). For instance, AC-Hunter recently added a new log file `open_conn.log`. If a user installs a plugin that generates a new log file or Zeek happens to start producing a new log we wouldn't be able to get that log without updating this script.

What if we just tested the string against a regex and only allowed `^[a-zA-Z0-9-_]+$`? I'm not aware of anything that could be abused path traversal-wise with only those characters and I'm willing to take the tradeoff of allowing more character combinations in exchange for being more future-proof.